convert_tuh_to_bids.py

```diff
@@ -15,19 +15,222 @@
 import re
 import argparse
 import datetime
+import glob
+import os
 
+from joblib import Parallel, delayed
+from pathlib import Path
 import mne
 import numpy as np
-from braindecode.datasets import TUHAbnormal
```

Comment:
I would be better to fix the braindecode code and use a fixed version rather than copy pasting all this code.

Reply:
+1 on that

Reply:
@agramfort It is a work in progress on the Braindecode repo [here](https://github.com/braindecode/braindecode/pull/431).